### ISSUE TITLE
Finite stuttering should be possible?

### ISSUE THREAD

**Issue by lemmy** [will62794/tla-web#11]
```tla
---- MODULE Foo ----

VARIABLE x

Init == x = TRUE

Next == x'\in BOOLEAN
====
```

<img width="670" alt="Screen Shot 2022-06-28 at 11 28 57 AM" src="https://user-images.githubusercontent.com/88777/176256244-3d37cb74-7602-49d8-bdf5-111e80c0b1ce.png">

**Comment by will62794** [comment]
I'm not sure exactly what issue you're referring to here, but generally I haven't yet put much thought into how to represent stuttering transitions in the interpreter and/or UI. Currently I [only consider transitions](https://github.com/will62794/tla-web/blob/62503af8aaf2e3790f36d34beaf9ef87152711ca/js/eval.js#L1942-L1945) that explicitly modify the state i.e. non-stuttering steps.

**Comment by lemmy** [comment]
The current behavior gets in the way when teaching TLA+.  Showing the stuttering step is also more consistent with the fix for issue #12.

**Comment by will62794** [comment]
Partial progress in 2734e6b.
